docs/content/en/hosting-and-deployment/hugo-deploy.md

```diff
@@ -85,8 +85,9 @@ cloudFrontDistributionID = <ID>
 # Optionally, you can include or exclude specific files.
 # See https://godoc.org/github.com/gobwas/glob#Glob for the glob pattern syntax.
 # If non-empty, the pattern is matched against the local path.
-# If exclude is non-empty, and a file's path matches it, that file is dropped.
-# If include is non-empty, and a file's path does not match it, that file is dropped.
+# If exclude is non-empty, and a file's path matches it, that file is not synced with the remote
+# If include is non-empty, and a file's path does not match it, that file is not synced with the remote
+# Files not synced with the remote will cause both local and remote copies of the file to remain intact should they exist.
```

Comment:
Hmm, I don't think these changes quite capture the semantics; "both local and remote" here implies that sometimes local files are deleted, which is never the case.

How about:

```
# All paths are matched against in their filepath.ToSlash form.
# If exclude is non-empty, and a local or remote file's path matches it, that file is not synced.
# If include is non-empty, and a local or remote file's path does not match it, that file is not synced.
# As a result, local files that don't pass the include/exclude filters are not uploaded to remote,
# and remote files that don't pass the include/exclude filters are not deleted.